Lib/test/test_repl.py

```diff
@@ -194,6 +194,10 @@ def bar(x):
         self.assertEqual(p.returncode, 0)
         expected = "(30, None, [\'def foo(x):\\n\', \'    return x + 1\\n\', \'\\n\'], \'<stdin>\')"
         self.assertIn(expected, output, expected)
+    
+    def test_asyncio_repl_is_ok(self):
+        assert_python_ok("-m", "asyncio")
```

Comment:
e created this test to check that running "asyncio REPL" is OK. Perhaps we could make it better, but I don't know how.